pkg/apis/options/options.go

```diff
@@ -166,6 +167,7 @@ func NewOptions() *Options {
 		ProxyWebSockets:     true,
 		HTTPAddress:         "127.0.0.1:4180",
 		HTTPSAddress:        ":443",
+		ProxyTimeOut:        time.Duration(30) * time.Second,
```

Comment:
This should work if done implicitly 
```suggestion
		ProxyTimeOut:        30 * time.Second,
```

Reply:
Ok